src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -58,10 +58,10 @@ def get_hostname_completion_list(prefix, action, parsed_args, **kwargs): # pylin
 register_cli_argument('appservice plan', 'admin_site_name', help='The name of the admin web app.')
 
 register_cli_argument('appservice web', 'slot', options_list=('--slot', '-s'), help="the name of the slot. Default to the productions slot if not specified")
-register_cli_argument('appservice web', 'name', configured_default='appservice/default_webapp_name',
+register_cli_argument('appservice web', 'name', configured_default='appservice/webapp',
```

Comment:
What do you think about calling this "appservice/web" in order to match the noun already used in the CLI? I like the consistency, and it feels a little more intuitive.